src/Traits/ModifierTrait.php

```diff
@@ -243,12 +243,31 @@ public function second($value)
      * Add years to the instance. Positive $value travel forward while
      * negative $value travel into the past.
      *
+     * When adding or subtracting years, if the resulting time is a date
+     * that does not exist, the result of this operation will always be the
+     * last day of the intended month.
+     *
```

Comment:
I would leave a suggestion, but I can't replace multi-line comments.

Suggestion:
If the new date does not exist, the last day of the month is used instead instead of overflowing into the next month.